frontend/src/components/common/authorized-menu/components/navigation-menu/helpers/get-permitted-subroutes.helper.ts

```diff
@@ -5,11 +5,13 @@ const getPermittedSubroutes = (
   subroutes: SubNavigationMenuItem[],
   user: UserWithPermissions,
 ): SubNavigationMenuItem[] => {
-  return subroutes.filter((route) =>
-    checkHasPermission({
-      permissionKeys: route.permissions,
-      userPermissions: user.permissions,
-    }),
+  return subroutes.filter(
+    (route) =>
+      !route.permissions.length ||
```

Comment:
This check should be in checkHasPermission helper

Reply:
Still isn't resolved, checkHasPermission should return true in case of empty array

Reply:
Still isn't resolved, checkHasPermission should return true in case of empty array

Reply:
But I added this check in new commit

Reply:
Oh, I see